### PR TITLE
Add owner name column option in report builder to case based reports

### PR DESCRIPTION
**Pull request by NoahCarnahan**
Addresses [Ticket #178054](http://manage.dimagi.com/default.asp?178054).
Add case owner name as an option for case based report columns.
@czue cc @dannyroberts

### CODE REVIEW

**Comment by czue** [comment]
no switch statement? :)

File: corehq/apps/userreports/reports/builder/forms.py [hunk 1]
```diff
@@ -177,6 +178,8 @@ def indicators(self):
                 ret.append(make_form_question_indicator(
                     prop['source'], prop['column_id']
                 ))
+            elif prop['type'] == 'case_property' and prop['source'] == 'computed/owner_name':
```
Comment (czue) [inline corehq/apps/userreports/reports/builder/forms.py]: seems fine for now but if we keep creeping this we should probably figure out a cleaner strategy for not hard coding a bunch of these

**Comment by czue** [comment]
:+1: from me me.

Would be nice to have some DB tests on the expression for the different owner types, but feel free to self merge if that feels like too much effort or poor ROI.

**Comment by czue** [comment]
@NoahCarnahan let me know if you want this merged

File: corehq/apps/userreports/reports/builder/forms.py (hunk 1 above)
Comment (NoahCarnahan) [inline corehq/apps/userreports/reports/builder/forms.py]: Agreed this isn't the cleanest. Will definitely revisit it if it gets any bigger.

**Pull request merged**